frontend/src/app/filter-blocks/filter-block/filter-block.component.ts

```diff
@@ -1,16 +1,44 @@
 import { Component, Input } from '@angular/core';
-import { Filter, FilterOperations } from '@common/types';
+import { Router, ActivatedRoute } from '@angular/router';
+import { Filter, FilterOperations, InputKeyTypes } from '@common/types';
+import { TranslateService } from '@ngx-translate/core';
+
+interface Key {
```

Comment:
Maybe rename to `DropdownOption` or something?

Reply:
done